crates/sui-indexer/src/store/pg_indexer_store.rs

```diff
@@ -370,6 +371,135 @@ impl IndexerStore for PgIndexerStore {
         }
     }
 
+    fn get_all_objects_page(
+        &self,
+        object_id: ObjectID,
+        limit: usize,
+        is_descending: bool,
+    ) -> Result<Vec<ObjectRead>, IndexerError> {
+        let mut pg_pool_conn = get_pg_pool_connection(&self.cp)?;
+
+        let objects: Vec<Object> = pg_pool_conn
+            .build_transaction()
+            .read_only()
+            .run(|conn| {
+                let boxed_query = objects_dsl::objects.into_boxed();
+
+                if is_descending {
+                    boxed_query
+                        .order(objects_dsl::object_id.desc())
+                        .filter(objects_dsl::object_id.lt(object_id.to_string()))
```

Comment:
we cannot use the `Order` of object string here as seq, but as mentioned above, this is prob the wrong table.